PendingReleaseNotes

```diff
@@ -182,3 +182,6 @@ Relevant tracker: https://tracker.ceph.com/issues/57090
   key/value set for a filesystem extended attributes.  It effectively replaces
   the old per-MDS `max_xattr_pairs_size` setting, which is now dropped.
   Relevant tracker: https://tracker.ceph.com/issues/55725
+* fs: two FS names can now be swapped, optionally along with their IDs, using
```

Comment:
nit: `fs:` isn't meaningful (developers use it for file systems, but we should try to avoid it in release notes). And, there needs to be an empty line after the last paragraph.

Reply:
Done.